connections/package.json

```diff
@@ -20,6 +20,9 @@
     "@angular/platform-browser": "^15.0.0",
     "@angular/platform-browser-dynamic": "^15.0.0",
     "@angular/router": "^15.0.0",
+    "@techiediaries/ngx-qrcode": "^9.1.0",
```

Comment:
@Siddharth-Tiwari1712 I like that you are testing and trying out two different ones, so we can see which works better

Reply:
Yeah i have used angularx-qrcode and techiediaries/ngx-qrcode.